dragonflybsd/vm/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ ! -f disk.qcow2 ]; then
+    echo "disk.qcow2 doesn't exists! Run setup.sh first."
+    exit 1
+fi
+
+echo "Running QEMU..."
+qemu-system-x86_64 -m 2048 \
+                   -drive if=virtio,file=disk.qcow2,format=qcow2 \
+                   -enable-kvm \
+                   -display gtk \
+                   -netdev user,id=mynet0,hostfwd=tcp:127.0.0.1:9272-:22 \
+                   -device virtio-net,netdev=mynet0 \
+                   -bios OVMF.fd \
```

Comment:
I think this binary isn't present in this directory, does the script work? We could add a line to the script that symlinks it or copy the binary here and just provide it in the repository (since we need the modified version with ESRT values)